src/table/toolbar/table-toolbar.component.ts

```diff
@@ -48,14 +48,12 @@ import { TableRowSize } from "../table.types";
 	template: `
 	<section
 		class="bx--table-toolbar"
-		[ngClass]="{'bx--table-toolbar--small' : size === 'sm'}">
+		[ngClass]="{'bx--table-toolbar--small' : size === 'sm'}"
+		[attr.aria-label]="actionBarLabel.subject | async"
+		role="group">
 		<div
-			*ngIf="model"
-			class="bx--batch-actions"
-			[ngClass]="{
-				'bx--batch-actions--active': selected
-			}"
-			[attr.aria-label]="actionBarLabel.subject | async">
+			*ngIf="model && selected"
+			class="bx--batch-actions bx--batch-actions--active">
```

Comment:
```suggestion
			*ngIf="model"
			class="bx--batch-actions"
			[ngClass]="{
				'bx--batch-actions--active': selected
			}">
```

We have animations that are applied when `bx--batch-actions--active`. When we remove the entire element when selected is false, this removes the animation.